extensions/wikia/AdEngine/js/template/bfab.js

```diff
@@ -1,24 +1,64 @@
 /*global define, require*/
 define('ext.wikia.adEngine.template.bfab', [
 	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.adEngine.video.videoAdFactory',
 	'wikia.log',
 	'wikia.document',
+	'wikia.window',
```

Comment:
Since we've got `'wikia.document'`, can't we just use `doc.window` instead of adding one more dependency? @bognix, @drets, @jacekjursza, @fraszczakszymon what do you think?